.github/workflows/pr-slack-notification.yaml

```diff
@@ -0,0 +1,296 @@
+name: Fetch All Open and Recently Closed Pull Requests using cron job
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 15 * * 1-5'  # At 15:30 on every day-of-week from Monday through Friday(UTC Time), 9pm in IST and 11:30 AM in EST.
+
+jobs:
+  fetch_all_pull_requests_and_notify_using_condition_and_cron:
+    runs-on: ubuntu-latest
+
+    env:
+      REPO: ${{ github.repository }}
+      BRANCH: main
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      API_URL: https://api.github.com/repos/redhat-developer/lsp4ij/pulls
+      SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+      WORKFLOW_BUILDER_WEBHOOK: ${{ secrets.WORKFLOW_BUILDER_WEBHOOK }}
+      NO_PR_WORKFLOW_BUILDER_WEBHOOK: ${{ secrets.NO_PR_WORKFLOW_BUILDER_WEBHOOK }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
@TrevCraw , I found Warnings with this checkout version `v2` and open an issue https://github.com/OpenLiberty/liberty-tools-intellij/issues/937

There are 3 occurrence in total in this file `pr-slack-notification.yaml` that need to update

Since we already opened this PR to main, do we need to wait for this feature branch to be merged into main, and then open another PR to main to resolve this issue?